gbdxtools/images/meta.py

```diff
@@ -143,6 +143,19 @@ def _rgb_bands(self):
     def _ndvi_bands(self):
         return [6, 4]
 
+    @property
+    def _ndwi_bands(self):
+        return [7, 0]
+
+    def ndwi(self, **kwargs):
+        """
+        Calculates Normalized Difference Water Index using bands 1 (Coastal) and 8 (NIR2) of a WV2, or WV3 image.
+
+        Returns: numpy array of ndwi values
+        """
+        data = self._read(self[self._ndwi_bands,...]).astype(np.float32)
+        return (data[0,:,:] - data[1,:,:]) / (data[0,:,:] + data[1,:,:])
```

Comment:
Here's where I suspect the indices are switched: I think we want `data[1, :, :] - data[0, :, :]` in the numerator... seem right to you?

Reply:
@martham93 Can you fix this?